deps/acorn/acorn-walk/package.json

```diff
@@ -4,7 +4,7 @@
   "homepage": "https://github.com/acornjs/acorn",
   "main": "dist/walk.js",
   "module": "dist/walk.mjs",
-  "version": "6.1.1",
+  "version": "6.3.1",
```

Comment:
The latest version is `6.1.1`, https://github.com/acornjs/acorn/blob/f164d84661f4f79486c84a4c0d1ca2233614c088/acorn-walk/package.json#L7.